uploadNeuroDB/minc_insertion.pl

```diff
@@ -335,29 +336,18 @@ =head2 Methods
     %studyInfo = $utility->createTarchiveArray($ArchiveLocation);
 
 } elsif ($tarchive) {
-    # if only the tarchive path is given as an argument, find the associated UploadID
-    # and check if IsTarchiveValidated is set to 1.
-    $ArchiveLocation = $tarchive;
-    $ArchiveLocation    =~ s/$tarchiveLibraryDir\/?//g;
-
-    # CONCAT ensures that ArchiveLocation always contains a slash at the beginning
-    my $query = "SELECT IsTarchiveValidated, UploadID, SourceLocation "
-                . "FROM mri_upload "
-                . "JOIN tarchive USING (TarchiveID) "
-                . "WHERE CONCAT('/', ArchiveLocation) LIKE ? ";
-    my $sth   = $dbh->prepare($query);
-    print $query . "\n" if $debug;
+    my $mriUploadOB = NeuroDB::objectBroker::MriUploadOB->new(db => $db);
+    my $mriUploadsRef = $mriUploadOB->getWithTarchive(GET_COLUMNS, $tarchive);
```

Comment:
why use a constant here instead of calling the function directly with `(0, $tarchive)`?

Reply:
I am not a big fan of having `0` as a parameter of a function when it is used as a boolean: I'd rather have a constant that indicates the intended use of that number.